CryptoPrices/Sources/Domain/Sources/Entities/CoinDetail/MarketData.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  MarketData.swift
+//
```

Comment:
Don't forget to add the package name - `Domain` here as well for the file's metadata  🙏 And please consider updating the rest 🚀
```suggestion
// Domain
//
```

Reply:
Good catch 🙌 . I almost forgot that. I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/78/commits/cf98fda5a3fa48421ee1278c9612ef324bfdc805.